addons/website_slides/static/src/js/portal_chatter.js

```diff
@@ -0,0 +1,27 @@
+odoo.define('website.slides.portal.chatter', function (require) {
+'use strict';
+
+var portalChatter = require('portal.chatter');
+
+var PortalChatter = portalChatter.PortalChatter;
+
+/**
+ * PortalChatter
+ *
+ * Extends Frontend Chatter to handle rating count on review tab
+ */
+PortalChatter.include({
+
+    /**
+     * Update review count on review tab in courses
+     * @override
+     * @private
+     */
+    _reloadChatterContent: async function (data) {
+        await this._super(...arguments);
+        if (this.options.res_model === "slide.channel") {
+            $('#review-tab').html('Reviews (' + data.rating_total + ')');
```

Comment:
Using html can lead to code injection, prefer text whenever possible.

Also this needs to be translatable.

Reply:
Updated using text, will be translatable.

Reply:
Updated using text, also string will be translatable now.